clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/handlers/BasicGoogleDeployHandler.groovy

```diff
@@ -159,7 +163,7 @@ class BasicGoogleDeployHandler implements DeployHandler<BasicGoogleDeployDescrip
                                                     metadata: metadata,
                                                     tags: tags,
                                                     scheduling: scheduling,
-                                                    serviceAccounts: [serviceAccount])
+                                                    serviceAccounts: serviceAccount ? [serviceAccount] : [])
```

Comment:
This was a misguided attempt to avoid ending up with a method named `buildServiceAccounts` that would seem to imply that we support more than one service account. I think you're right though. I'll change it and just leave the name `buildServiceAccount`.